public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -119,6 +132,21 @@ export const LokiQueryBuilderOptions = React.memo<Props>(
               aria-label="Select resolution"
             />
           </EditorField>
+          {config.featureToggles.lokiQuerySplitting && (
+            <EditorField
+              label="Chunk Range"
+              tooltip="Experimental: Defines the range of a single query chunk when query chunking is used."
```

Comment:
```suggestion
              tooltip="Defines the range of a single query chunk when query splitting is used."
```

I need a native speaker to back me up, but "chunking" sounds awful to me.

Reply:
It seems that we're rolling with chunking everywhere, so no need to change it.